### DISCUSSION TITLE
[Projects Beta] Add option to change color in Single Select values

### DISCUSSION THREAD

**Discussion by oscard0m** [community/community#7064] [Projects and Issues]
EDIT:

Duplicated: https://github.com/github/feedback/discussions/6234

---

It would be nice to have the option to change the color to values under Single Select fields (as issue labels have the option):

![image](https://user-images.githubusercontent.com/2574275/139574570-1a1b352c-00e5-4272-9dc4-42c9513b6e3a.png)

I guess this is something for future iterations since this is a BETA but I just wanted to pop it up just in case :)

**Comment by Katsute** [comment]
https://github.com/github/feedback/discussions/6234

**Reply by oscard0m** [reply]
Thanks @Katsute .

I agree with [this comment](https://github.com/github/feedback/discussions/6234#discussioncomment-1483313) that it helps but it would be nice to get actual background color as labels do.
